docs/superpowers/plans/2026-08-04-tabbar-minimize-player-squeeze.md

```
@@ -0,0 +1,259 @@
+    .accentColor(.white)  // Makes the selected tab icon white
+    .playolaTabBarChrome()
+    .playolaTabBarMinimize()
+    .playolaBottomAccessory(isEnabled: model.shouldShowSmallPlayer) {
+      smallPlayer(isGlassAccessory: true)
+    }
+```
+
+- [ ] **Step 4: Build to verify compilation on the CI SDK (and confirm availability isolation)**
+
+Run:
+```bash
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)